index.html

```diff
@@ -688,6 +689,11 @@
         when requesting the sending of a <a>push message</a> from the <a>push service</a>.
       </p>
       <p>
+        The <dfn data-dfn-for="PushManager">supportedAuthenticationSchemes</dfn> attribute exposes
+        the sequence of supported authentication schemes that can be used by <a>applications
+        servers</a> to authenticate with the <a>push service</a>.
+      </p>
```

Comment:
Also, does `vapid` refer to the latest version of the draft, or the previous one (still using `Crypto-Key`) that still has more implementations?